.github/workflows/integration.yml

```
@@ -70,6 +70,18 @@ jobs:
           retention-days: 5
           if-no-files-found: error
 
+  types:
```

This new job is not required, it's just an extra signal that makes the failure reason clearer (as opposed to "some error in webpack OR in the types")

Feel free to include this in the code as well:

```suggestion
  // Surface type errors in GitHub’s UI faster and outside webpack
  types:
```

---

Speaking of better CI experience, it looks like the `upload-artifact` takes 4 minutes to complete. Can we disable it on PRs or is it useful there too? We can leave it on `main`, tags and `release/*`, thoughts?

---

> Can we disable it on PRs or is it useful there too? We can leave it on main, tags 
> and release/*, thoughts?

Let's go for it 👍